packages/prisma/migrations/20220803091114_drop_daily_event_reference/migration.sql

```diff
@@ -9,3 +9,6 @@ ALTER TABLE "DailyEventReference" DROP CONSTRAINT "DailyEventReference_bookingId
 
 -- DropTable
 DROP TABLE "DailyEventReference";
+
```

Comment:
Please be aware that editing an already applied migration can be [potentially dangerous](https://www.prisma.io/docs/concepts/components/prisma-migrate#do-not-edit-or-delete-migrations-that-have-been-applied) 🌻

Reply:
Yep @simobasso I think we should probably create a new migration that fixes the old one right?

Reply:
> Please be aware that editing an already applied migration can be [potentially dangerous](https://www.prisma.io/docs/concepts/components/prisma-migrate#do-not-edit-or-delete-migrations-that-have-been-applied) 🌻

- If the migration was already applied (for postgres >12) Changing the migration won't affect them since it will be already marked as applied
- If the migration failed (<11), changing the migration won't affect them since they can try apply the migration again.

@CarinaWolli just followed my suggestion of updating the migrations based on my points above.